colossalai/kernel/cuda_native/csrc/attention_infer_kernels/softmax/fused_softmax.cpp

```diff
@@ -0,0 +1,46 @@
+/* coding=utf-8
+ * Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Is it from apex? Can we use it directly?

Reply:
it is not from apex, it is from faster transformer

Reply:
currently, I saw there already has some scaled softmax in kernels, I can temporarily remove this. If I feel the original one is not useful, I can add it back.